scripts/experimental/install_dev_osgeo.sh

```diff
@@ -71,16 +71,16 @@ LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 if [ "$GEOS_VERSION" = "latest" ]; then
     GEOS_VERSION=$(wget -qO- "https://api.github.com/repos/libgeos/geos/git/refs/tags" | grep -oP "(?<=\"ref\":\s\"refs/tags/)\d+\.\d+\.\d+" | tail -n -1)
 fi
-
-wget http://download.osgeo.org/geos/geos-"${GEOS_VERSION}".tar.bz2
+wget -q http://download.osgeo.org/geos/geos-${GEOS_VERSION}.tar.bz2
```

Comment:
Is this change necessary? (I believe it needs to be enclosed in at least double quotes)